subprojects/docs/src/samples/userguide/java/basic/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,106 @@
+// tag::apply-java-plugin[]
+plugins {
+    java
+}
+
+project.setProperty("sourceCompatibility", "1.8")
+project.setProperty("targetCompatibility", "1.8")
+
+version = "1.2.1"
+// end::apply-java-plugin[]
+
+// tag::java-dependency-mgmt[]
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.hibernate:hibernate-core:3.6.7.Final")
+}
+// end::java-dependency-mgmt[]
+
+// tag::java-basic-test-config[]
+dependencies {
+    testImplementation("junit:junit:4.12")
+}
+
+tasks.getByName<Test>("test") {
+    useJUnit()
+
+    maxHeapSize = "1G"
+}
+// end::java-basic-test-config[]
+
+// tag::practical-integ-test-source-set[]
+sourceSets {
+    create("intTest") {
+        compileClasspath += sourceSets["main"].output
+        runtimeClasspath += sourceSets["main"].output
+    }
+}
+
+val intTestImplementation by configurations.getting {
```

Comment:
`val intTestImplementation = configurations.getByName("intTestImplementation") { ... }` I think would be clearer. 

Unfortunately `val intTestImplementation = configurations["intTestImplementation"]` "cannot be invoked as a function". Is this a bug @eskatos?

We want to avoid unnecessary use of delegated properties in these samples. That doesn't mean it's forbidden just only when it's necessary. The goal here is to spare non-Kotlin users too Kotlin-y syntax.